### PR TITLE
Upgrade

### PR DESCRIPTION
**Pull request by iArisu**

### CODE REVIEW

**Comment by iArisu** [comment]
I hope it's understandable and everything works :)

File: js/jquery-3.6.0.min.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
...
...)},hover:function(e,t){return this.mouseenter(e).mouseleave(t||e)}}),S.each("blur focus focusin focusout resize scroll click dblclick mousedown mouseup mousemove mouseover mouseout mouseenter mouseleave change select submit keydown keypress keyup contextmenu".split(" "),function(e,n){S.fn[n]=function(e,t){return 0<arguments.length?this.on(n,null,e,t):this.trigger(n)}});var Xt=/^[\s\uFEFF\xA0]+|[\s\uFEFF\xA0]+$/g;S.proxy=function(e,t){var n,r,i;if("string"==typeof t&&(n=e[t],t=e,e=n),m(e))return r=s.call(arguments,2),(i=function(){return e.apply(t||this,r.concat(s.call(arguments)))}).guid=e.guid=e.guid||S.guid++,i},S.holdReady=function(e){e?S.readyWait++:S.ready(!0)},S.isArray=Array.isArray,S.parseJSON=JSON.parse,S.nodeName=A,S.isFunction=m,S.isWindow=x,S.camelCase=X,S.type=w,S.now=Date.now,S.isNumeric=function(e){var t=S.type(e);return("number"===t||"string"===t)&&!isNaN(e-parseFloat(e))},S.trim=function(e){return null==e?"":(e+"").replace(Xt,"")},"function"==typeof define&&define.amd&&define("jquery",[],function(){return S});var Vt=C.jQuery,Gt=C.$;return S.noConflict=function(e){return C.$===S&&(C.$=Gt),e&&C.jQuery===S&&(C.jQuery=Vt),S},"undefined"==typeof e&&(C.jQuery=C.$=S),S});
```
[the remark this answers is not in the document]
Comment (iArisu) [reply js/jquery-3.6.0.min.js]: locally removed jquery

[the remark this answers is not in the document]
Comment (iArisu) [reply js/jquery-3.6.0.min.js]: locally removed jquery

[the remark this answers is not in the document]
Comment (iArisu) [reply js/jquery-3.6.0.min.js]: locally removed jquery

[the remark this answers is not in the document]
Comment (iArisu) [reply js/jquery-3.6.0.min.js]: locally removed jquery

[the remark this answers is not in the document]
Comment (iArisu) [reply js/jquery-3.6.0.min.js]: locally removed jquery

[the remark this answers is not in the document]
Comment (iArisu) [reply js/jquery-3.6.0.min.js]: locally removed jquery

[the remark this answers is not in the document]
Comment (iArisu) [reply js/jquery-3.6.0.min.js]: locally removed jquery

[the remark this answers is not in the document]
Comment (iArisu) [reply js/jquery-3.6.0.min.js]: locally removed jquery

[the remark this answers is not in the document]
Comment (iArisu) [reply js/jquery-3.6.0.min.js]: locally removed jquery

File: js/main.js [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+                // Handle the back (or forward) buttons here
+                // Will NOT handle refresh, use onbeforeunload for this.
+                goBack();
+            };
+        } else {
+            var ignoreHashChange = true;
+            window.onhashchange = function () {
+                if (!ignoreHashChange) {
+                    ignoreHashChange = true;
+                    window.location.hash = Math.random();
```
[the remark this answers is not in the document]
Comment (iArisu) [reply js/main.js]: Removed unused code

**Comment by FlowArg** [comment]
Could you replace jquery by just dom please? I don't want that shit here.
Ensure the PR id valid because I can't rebase & merge it.

**Comment by iArisu** [comment]
> Could you replace jquery by just dom please? I don't want that shit here.
> Ensure the PR id valid because I can't rebase & merge it.

Ok... Why do you don't want jQuery ?

**Comment by iArisu** [comment]
Ps : I don't know how to see/fix conflicts

**Comment by FlowArg** [comment]
> Ok... Why do you don't want jQuery ?

outdated, useless atm, decrease performance

**Comment by iArisu** [comment]
> > Ok... Why do you don't want jQuery ?
> 
> outdated, useless atm, decrease performance

Ok

**Comment by iArisu** [comment]
Nice !
_PS: Don't forget when you to add files to add them to `./js/main.js` -> `avaible_paths`_

**Pull request merged**